### PR TITLE
Added screenshots for epam integration

### LINKED ISSUE #4829: [Screenshots] Epam Delivery Platform (EDP) Admin Console under "How it Works See It in Action” section
### Current Behavior
Currently, [every integration](https://layer5.io/cloud-native-management/meshery/integrations/epam-delivery-platform-edp-admin-console) has only two old screenshots of the designer and visualizer in the "How it Works See It in Action” section.

### Desired Behavior
- Add new images and videos from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap))
- Add screenshots for [Epam Delivery Platform (EDP) Admin Console](https://layer5.io/cloud-native-management/meshery/integrations/epam-delivery-platform-edp-admin-console) integration.

### Acceptance Tests
- Ensure your screenshot represents the use of a few specific integrations.
- The screenshot from the Designer mode should contain a valid design, refer to [meshery catalog](https://meshery.io/catalog) to use and view published designs.
- Add both Dark Mode and Light Mode screenshots.
- The screenshot should not be a snippet, take the full screenshot of your browser.
- Use Incognito mode on Chrome browser, to ensure your bookmark bar or any other items are not shown in the screenshot/video.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by ankushbhardwaj408**
**Description**

This PR fixes #4829 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit fac9a58fd4a3639a2603780f94d44fa7c1e9c8a4 at: https://65244380f2a8d71ccddd98c8--layer5.netlify.app

**Review by sandramsc — CHANGES_REQUESTED** [review]
Hi, thank you for your contribution, please create new screenshot for visualizer as icons are missing from the canvas. View the comment on this [PR](https://github.com/layer5io/layer5/pull/4914#issuecomment-1734386234) for more context.

**Comment by ankushbhardwaj408** [comment]
Sure @sandramsc i'll work on it.

**Comment by l5io** [comment]
🚀 Preview for commit 5759ab88e25b16ad035795b603f8418f86b2b46f at: https://6527fbd6dbaa8e1b51cd2e72--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 7da025b7c2b6098cb67e856e2fb06cb4678d667e at: https://652c196f0ec6f612db66d736--layer5.netlify.app

**Comment by saurabh100ni** [comment]
Thanks for raising PR, Let's discuss this in the Website call today at 5:30 PM IST.
Please add this as your agenda in [Docs](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk)

**Comment by l5io** [comment]
🚀 Preview for commit 68269f6e6cb28c97c4991bd0a08c19c16bc684fa at: https://652f887ea49f01639882b5ac--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4a1489a98b5f6f1d271725493fa78fe4ed842320 at: https://6532050afd8e6c55bca590d4--layer5.netlify.app

**Comment by ankushbhardwaj408** [comment]
@sandramsc please review this pr.

**Comment by ankushbhardwaj408** [comment]
> Hi @ankushbhardwaj408 , thank you for your contribution, please create new screenshot for visualizer as icons are missing from the canvas. View the comment on this [PR](https://github.com/layer5io/layer5/pull/4914#issuecomment-1734386234) for more context.

@sandramsc please review this pr now.

**Comment by l5io** [comment]
🚀 Preview for commit 4f74bc99690dc41263d86630542e626768d191bc at: https://653d23bd8abfc044241b810f--layer5.netlify.app

**Comment by Mohith234** [comment]
@ankushbhardwaj408 Let's discuss this on website call today at 5:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

**Comment by l5io** [comment]
🚀 Preview for commit 178862a13ee1caf474d2649136a7c60a4c47cd5e at: https://65461539b9207a663c40abe1--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@ankushbhardwaj408 merge conflict

**Comment by ankushbhardwaj408** [comment]
@sudhanshutech merge conflicts resolved.

**Comment by l5io** [comment]
🚀 Preview for commit 8f0a5845063c1659410df82896840cbf432d6a51 at: https://655251c3f309090077bfba1a--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8c75a4dbc89818561401337d670f5bb59b1d8e2c at: https://6553812b9d19d20c028ef8b9--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3d99681d910b949b0756503f35849282a796ee43 at: https://6555968b6c27593263ec5b9d--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c4d367ba5cb75f12fc031b2c4e4fd56b4bacf9a1 at: https://655ddcc5036f1f0e39c2ee7d--layer5.netlify.app

**Comment by ankushbhardwaj408** [comment]
https://655ddcc5036f1f0e39c2ee7d--layer5.netlify.app/cloud-native-management/meshery/integrations/epam-delivery-platform-edp-admin-console

**Comment by ankushbhardwaj408** [comment]
https://655ddcc5036f1f0e39c2ee7d--layer5.netlify.app/cloud-native-management/meshery/integrations/epam-delivery-platform-edp-admin-console

**Comment by leecalcote** [comment]
@ankushbhardwaj408 I believe that @vishalvivekm is asking for changes.

**Comment by ankushbhardwaj408** [comment]
@leecalcote i was out of town. I'll get it done asap.

**Comment by l5io** [comment]
🚀 Preview for commit ee1f9a45b2068bff78fa2f6dc07c92affa2be7bf at: https://656cbb35cb669e290e5bc1a2--layer5.netlify.app

**Comment by ankushbhardwaj408** [comment]
@leecalcote i am not able to commit new changes 
Screenshot 2023-12-04 083449
Screenshot 2023-12-04 083420
I think this is due to embed tests in static folder. I have deleted the entire repo and restarted the local server but still this error is appearing while commiting

**Comment by vishalvivekm** [comment]
> @leecalcote @vishalvivekm i am not able to commit new changes Screenshot 2023-12-04 083420 Screenshot 2023-12-04 083449

Thanks for reporting this @ankushbhardwaj408.

**Comment by ankushbhardwaj408** [comment]
@vishalvivekm ??

File: src/collections/integrations/_images/admin-console-operator/meshmap-epam-designer-dark.png
Comment (vishalvivekm) [inline src/collections/integrations/_images/admin-console-operator/meshmap-epam-designer-dark.png:1]: @ankushbhardwaj408 the folder in _images/ has to be named `epam-delivery-platform-edp-admin-console`

Here the title is : Epam Delivery Platform (EDP) Admin Console
its sluggified versoin: `epam-delivery-platform-edp-admin-console`

: https://github.com/layer5io/layer5/pull/5129#issue-1995493031

Comment (ankushbhardwaj408) [reply src/collections/integrations/_images/admin-console-operator/meshmap-epam-designer-dark.png:1]: Will change it🫡

**Comment by l5io** [comment]
🚀 Preview for commit e8c9a12da81efe070cd3ab16b5a4db8448d72f38 at: https://656de5f3eb29fa00871c3719--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 214838a5ce0fa81b3f529e64e38ef865ff96fd31 at: https://656ded6f65c6b305ac017999--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit a0506648b622c02c6f6117b0b18ed3332c25bcd6 at: https://656e1427fa1051009a473201--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 9083133da30d9596a827b3a16daeb6603dd62989 at: https://656e1bba569df9008c20047a--layer5.netlify.app

**Review by vishalvivekm — APPROVED** [review]
LGTM.
Thank you @ankushbhardwaj408

**Pull request merged**